packages/block-editor/src/components/button-block-appender/style.scss

```diff
@@ -23,16 +23,27 @@
 
 	// This variant is used in inline situations, like Buttons, Social Links, Navigation Menu.
 	&.block-list-appender__toggle {
-		background: $dark-gray-primary;
-		color: $white;
+		display: flex;
+		flex-direction: row;
+		color: $dark-gray-primary;
 		box-shadow: none;
-		width: 24px;
 		height: 24px;
 		padding: 0;
 		margin-left: $grid-unit-10;
 
 		&:active {
 			color: $white;
 		}
+
+		& > svg {
+			width: 24px;
+			background-color: $dark-gray-primary;
+			color: $white;
+		}
+
+		& > span {
```

Comment:
Can we be more explicit with selectors. The increasing chain or non-class selectors and the descendant selector makes the code a bit more fragile. I'd add a class name to the span.